inc/Engine/Optimization/DelayJS/Admin/Settings.php

```diff
@@ -217,15 +228,14 @@ public static function exclusion_list_has_default(): bool {
 	 * @return void
 	 */
 	public function refresh_exclusions_option() {
-		$current_selected_exclusions_selected_exclusion = get_rocket_option( 'delay_js_exclusions_selected_exclusions', '' );
-
-		if ( empty( $current_selected_exclusions_selected_exclusion ) ) {
-			update_rocket_option( 'delay_js_exclusions_selected_exclusions', '' );
-
+		$selected_items = $this->option->get( 'delay_js_exclusions_selected' );
+		if ( empty( $selected_items ) ) {
+			$this->option->set( 'delay_js_exclusions_selected_exclusions', [] );
```

Comment:
```suggestion
			$this->options->set( 'delay_js_exclusions_selected_exclusions', [] );
```

The Options_Data is not enough, we also need the options api class instance to save the settings